Panic error when telegraf starts

## Bug report

### System info:

telegraf version: 0.13.1
system: red hat 5.2

### Steps to reproduce:

1. Start telegraf


### Expected behavior:

### Actual behavior:
The following error:
2016/06/03 11:04:43 Starting Telegraf (version 0.13.1)
2016/06/03 11:04:43 Loaded outputs: influxdb
2016/06/03 11:04:43 Loaded inputs: disk net kernel mem processes swap system cpu diskio
2016/06/03 11:04:43 Tags enabled: host=cw-spmi-oas05
2016/06/03 11:04:43 Agent Config: Interval:10s, Debug:false, Quiet:false, Hostname:"cw-spmi-oas05", Flush Interval:10s 
panic: runtime error: index out of range

goroutine 69 [running]:
panic(0x107ebc0, 0xc82000e080)
	/usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/shirou/gopsutil/disk.IOCounters(0x0, 0x0, 0x0)
	/home/ubuntu/telegraf-build/src/github.com/shirou/gopsutil/disk/disk_linux.go:299 +0x823
github.com/influxdata/telegraf/plugins/inputs/system.(*systemPS).DiskIO(0x1c78550, 0x0, 0x0, 0x0)
	/home/ubuntu/telegraf-build/src/github.com/influxdata/telegraf/plugins/inputs/system/ps.go:118 +0x28
github.com/influxdata/telegraf/plugins/inputs/system.(*DiskIOStats).Gather(0xc8200116e0, 0x2aaaaac8b638, 0xc820132330, 0x0, 0x0)
	/home/ubuntu/telegraf-build/src/github.com/influxdata/telegraf/plugins/inputs/system/disk.go:104 +0x67
github.com/influxdata/telegraf/agent.gatherWithTimeout.func1(0xc8203be480, 0xc820011740, 0xc820132330)
	/home/ubuntu/telegraf-build/src/github.com/influxdata/telegraf/agent/agent.go:174 +0x73
created by github.com/influxdata/telegraf/agent.gatherWithTimeout
	/home/ubuntu/telegraf-build/src/github.com/influxdata/telegraf/agent/agent.go:175 +0xe0

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -49,6 +49,7 @@ time before a new metric is included by the plugin.
 - [#1297](https://github.com/influxdata/telegraf/issues/1297): disk plugin panic when usage grab fails.
 - [#1316](https://github.com/influxdata/telegraf/pull/1316): Removed leaked "database" tag on redis metrics. Thanks @PierreF!
 - [#1323](https://github.com/influxdata/telegraf/issues/1323): Processes plugin: fix potential error with /proc/net/stat directory.
+- [#1322](https://github.com/influxdata/telegraf/issues/1322): Fix rare RHEL 5.2 panic in gopsutil diskio gathering function.
 
 ## v0.13.1 [2016-05-24]
 

--- a/Godeps
+++ b/Godeps
@@ -43,7 +43,7 @@ github.com/prometheus/client_model fa8ad6fec33561be4280a8f0514318c79d7f6cb6
 github.com/prometheus/common e8eabff8812b05acf522b45fdcd725a785188e37
 github.com/prometheus/procfs 406e5b7bfd8201a36e2bb5f7bdae0b03380c2ce8
 github.com/samuel/go-zookeeper 218e9c81c0dd8b3b18172b2bbfad92cc7d6db55f
-github.com/shirou/gopsutil 83c6e72cbdef6e8ada934549abf700ff0ba96776
+github.com/shirou/gopsutil 586bb697f3ec9f8ec08ffefe18f521a64534037c
 github.com/soniah/gosnmp b1b4f885b12c5dcbd021c5cee1c904110de6db7d
 github.com/streadway/amqp b4f3ceab0337f013208d31348b578d83c0064744
 github.com/stretchr/testify 1f4a1643a57e798696635ea4c126e9127adb7d3c

